Adjust Vert.x paths for running
Index: vert.x-2.1M2/bin/vertx
===================================================================
--- vert.x-2.1M2.orig/bin/vertx	2013-12-01 10:02:43.000000000 -0300
+++ vert.x-2.1M2/bin/vertx	2014-01-09 19:18:16.495434205 -0300
@@ -55,25 +55,9 @@
     [ -n "$JAVA_HOME" ] && JAVA_HOME=`cygpath --unix "$JAVA_HOME"`
 fi
 
-# Attempt to set VERTX_HOME
-# Resolve links: $0 may be a link
-PRG="$0"
-# Need this for relative symlinks.
-while [ -h "$PRG" ] ; do
-    ls=`ls -ld "$PRG"`
-    link=`expr "$ls" : '.*-> \(.*\)$'`
-    if expr "$link" : '/.*' > /dev/null; then
-        PRG="$link"
-    else
-        PRG=`dirname "$PRG"`"/$link"
-    fi
-done
-SAVED="`pwd`"
-cd "`dirname \"$PRG\"`/.."
-VERTX_HOME="`pwd -P`"
-cd "$SAVED"
-
-CLASSPATH=${CLASSPATH}:${VERTX_HOME}/conf:${VERTX_HOME}/lib/*
+LIBDIR=/usr/share/vertx
+CONFDIR=/etc/vertx
+CLASSPATH=${CLASSPATH}:${CONFDIR}/:${LIBDIR}/*
 
 # Determine the Java command to use to start the JVM.
 if [ -n "$JAVA_HOME" ] ; then
@@ -174,8 +158,7 @@
 
 exec "$JAVACMD" \
     "${JVM_OPTS[@]}" \
-    -Djava.util.logging.config.file=${VERTX_JUL_CONFIG:-${VERTX_HOME}/conf/logging.properties} \
-    -Dvertx.home=$VERTX_HOME\
+    -Djava.util.logging.config.file=${CONFDIR}/logging.properties \
     -Dvertx.clusterManagerFactory=org.vertx.java.spi.cluster.impl.hazelcast.HazelcastClusterManagerFactory\
     -classpath "$CLASSPATH" \
     org.vertx.java.platform.impl.cli.Starter "$@"
