Adjust Vert.x paths for running
--- a/bin/vertx
+++ b/bin/vertx
@@ -65,7 +65,11 @@
 APP_HOME="`pwd -P`"
 cd "$SAVED"
 
-CLASSPATH=$APP_HOME/lib/*:$APP_HOME/conf:$JYTHON_HOME/jython.jar::$JRUBY_HOME/lib/jruby.jar
+JRUBY_HOME=/usr/lib/jruby
+LIBDIR=/usr/share/vertx
+CONFDIR=/etc/vertx
+
+CLASSPATH=$LIBDIR/*:$CONFDIR/conf:/usr/share/java/jython.jar:$JRUBY_HOME/lib/jruby.jar
 
 # Determine the Java command to use to start the JVM.
 if [ -n "$JAVA_HOME" ] ; then
@@ -164,4 +168,4 @@
 }
 eval splitJvmOpts $DEFAULT_JVM_OPTS $JAVA_OPTS $VERTX_OPTS
 
-exec "$JAVACMD" "${JVM_OPTS[@]}" -Dvertx.install=$APP_HOME -Djava.util.logging.config.file=$APP_HOME/conf/logging.properties -Djruby.home=$JRUBY_HOME -classpath "$CLASSPATH" org.vertx.java.deploy.impl.cli.Starter "$@"
+exec "$JAVACMD" "${JVM_OPTS[@]}" -Djava.util.logging.config.file=$CONFDIR/logging.properties -Djruby.home=$JRUBY_HOME -classpath "$CLASSPATH" org.vertx.java.deploy.impl.cli.Starter "$@"
